content/summits/developer/2025-Autumn/_index.md

```diff
@@ -0,0 +1,175 @@
+}
+</style>
+
+{{< details "**Tuesday**" >}}
+| Time | Description |
+|------|-------------|
+| | Arrive |
+| 18:30 | Informal group dinner (meet in lobby) |
+{{< /details >}}
+
+{{< details "**Wednesday**" >}}
+| Time | Description |
+|------|-------------|
+| 9:00--9:30 | Breakfast |
+| 9:00--9:30 | Welcome & coordination |
+| 9:30--13:00 | Work session |
+| 13:00--14:30 | Lunch & campus/coffee walks |
+| 14:30--17:30 | Work session |
+| 17:30--18:00 | Check-in / discussion |
+{{< /details >}}
+
+{{< details "**Thursday**" >}}
+| Time | Description |
+|------|-------------|
+| 9:00--9:30 | Breakfast |
+| 9:00--9:30 | Welcome & coordination |
+| 9:30--13:00 | Work session |
+| 13:00--14:30 | Lunch & campus/coffee walks |
+| 14:30--17:30 | Work session |
+| 17:30--18:00 | Check-in / discussion |
+{{< /details >}}
+
+{{< details "**Friday**" >}}
+| Time | Description |
+|------|-------------|
+| 9:00--9:30 | Breakfast |
+| 9:00--9:30 | Welcome & coordination |
+| 9:30--13:00 | Work session |
+| 13:00--14:30 | Lunch & campus/coffee walks |
+| 14:30--17:30 | Work session |
+| 17:30--18:00 | Check-in / discussion |
+| 19:00 | Dinner (Big Time Brewery) |
+{{< /details >}}
+
+{{< details "**Saturday**" >}}
+| Time | Description |
+|------|-------------|
+| | Depart |
+{{< /details >}}
+
+</div>
+
+The event is held at ...
+
+## Funding
+
+Unfortunately, this year we do not have any dedicated funding to cover participants travel and accommodation costs.
+
+Project maintainers, please reach out to your individual projects or employers to determine whether they have funds to sponsor your attendance.
+
+
+## Logistics
+
+We have space for 25 participants for three days and four nights.
+This will be an invite-only event that requires upfront agreement to:
+(a) take part in at least one pre-summit planning meeting,
+(b) collaborate with fellow participants on a work plan,
+(c) attend the summit in-person, and
+(d) participate, to whatever degree possible, in several months of post-summit implementation.
+
+### Airport to hotel
+
+...
+
+### Food
+
+...
+
+### Participants
+
+Participants are recruited from the community of developers of packages
+such as NumPy, SciPy, matplotlib, xarray, pandas, scikit-image, scikit-learn,
+NetworkX, and IPython, as well as domain stacks including Astropy, Pangeo, and
+scikit-HEP.
+
+{{< details "**List of confirmed participants**" >}}
+
+- ...
+
+{{< /details >}}
+
+## About
+
+### Work Topics
+
+The 2025 Autumn Scientific Python Developer Summit will focus on the following work topics:
+
+- **Data analysis**: statistics, optimization, sparse data, plotting, and dataframes.
+- **SPECS**: Community & governance SPECs, new SPECs, infrastructure & tools, etc.
+- **Tools**: CI, build, testing, cookie-cutter, new tools (https://tools.scientific-python.org), etc.
+- **Communications & teaching**: devstats, mystmd, lecture notes & learn
+- **Security**: supply chain, pull request review, tooling & infrastructure, web of trust
```

Comment:
This is the important bit, I think we should come up with a concise list of focus topics.